seahorn/lib/linked_list_helper.c

```diff
@@ -64,89 +27,259 @@ static inline void init_short_aws_linked_list(struct aws_linked_list *list,
   }
 }
 
-void sea_nd_init_aws_linked_list_from_front(struct aws_linked_list *list,
-                                            size_t *length) {
+// if nd_voidp() is assigned to next, prev without assigning them sane
+// values, then compiler treats them as undefined.
+void initNode(struct aws_linked_list_node *node) {
```

Comment:
would be nice if all functions follow the same naming style

Reply:
done